website/src/docs/hotchocolate/defining-a-schema/operations.md

```diff
@@ -279,38 +239,38 @@ public class Startup
 <ExampleTabs.Schema>
 
 ```csharp
-// Mutation.cs
 public class Mutation
 {
-    public async Task<Book> AddNewBook(Book book)
+    public async Task<Book> AddBook(Book book)
     {
         // Omitted code for brevity
     }
 }
 
-// Startup.cs
 public class Startup
 {
     public void ConfigureServices(IServiceCollection services)
     {
         services
-            .AddRouting()
             .AddGraphQLServer()
             .AddDocumentFromString(@"
-                type Query {
-                  book: Book
-                }
+                # ...
 
                 type Mutation {
-                  addNewBook(input: AddNewBookInput): AddNewBookPayload
+                  addBook(input: BookInput): Book
+                }
+
+                type BookInput {
```

Comment:
I think we should showcase how we would bind the BookInput to the Book CLR type? I tried `BindComplexType<Book>(c => c.To("BookInput"))` but had no success. 

How would I do it in this case?

Reply:
I will check.